managedtenants/bundles/index_builder.py

```diff
@@ -60,7 +68,14 @@ def _build(self, bundles, hash_string, skip_validation=False):
             "add",
             "--permissive",
             "--bundles",
-            ",".join([bundle.image.url_tag for bundle in bundles]),
+            ",".join(
+                [
+                    bundle.image.url_digest
+                    if not test_run
```

Comment:
why do we not want to resolve the digest during testing?

Reply:
because the current testing registry does not provide any metadata i.e. the manifest that we require here: https://github.com/app-sre/sretoolbox/blob/86d49c7ac4d27aaf3dea4f179d2f02247285f120/sretoolbox/container/image.py#L123-L146

Reply:
update: On digging into the tests we have in place, I found that the bundles weren't getting pushed in the test registry. Logs:
```
INFO     managedtenants-bundle-builder:bundle_builder.py:68 Pushing Bundle(addon_name=reference-addon, operator_name=reference-addon, version=0.1.6, path=/home/prraj/Desktop/redhat-projects/managed-tenants-cli/tests/testdata/addons/reference-addon/main/0.1.6, image=127.0.0.1:9999/reference-addon-bundle:0.1.6-0aa5ea5d8c3c4d1989d3d91ae3048e41).
DEBUG    managedtenants-docker:docker_api.py:120 {'status': 'The push refers to repository [127.0.0.1:9999/reference-addon-addon-operator-bundle]'}
DEBUG    managedtenants-docker:docker_api.py:120 {'errorDetail': {'message': 'Get "http://127.0.0.1:9999/v2/": read tcp 127.0.0.1:33606->127.0.0.1:9999: read: connection reset by peer'}, 'error': 'Get "http://127.0.0.1:9999/v2/": read tcp 127.0.0.1:33606->127.0.0.1:9999: read: connection reset by peer'}
```
Working on fixing this too.

Reply:
This is fixed as we use quay.io/mtsre/registry as the `LocalDockerRegistry` now.